### ISSUE TITLE
Tempo Tapper

### ISSUE THREAD

**Issue by Captain-Quack** [musescore/MuseScore#19037]
### Your idea

The tempo tapper would record a specific button (e.g., spacebar, left click) for 10 seconds or so, and set the bpm to the rate of which the user clicked the button. While there are already [existing versions of this online](https://tunebat.com/BPM-Tempo-Tapper), but it would be nice to have this builtin into musescore. 

P.S., the tempo tapper from the [osu](https://osu.ppy.sh/home) map creator works like a charm, it even allows you to offset a beat by a note or two! 

### Problem to be solved

Allowing creators to quickly configure the tempo of their song without having to do a lot of work.

**Comment by XiaoMigros** [comment]
There are plugins that can do this already, for example https://musescore.org/en/project/tap-bpm

**Comment by lgvr123** [comment]
And [Tap tempo](https://github.com/lgvr123/musescore-taptempo) where you can select to which type of note it applies.

**Comment by joexcm** [comment]
I agree this should be built into the new score wizard. I'm constantly going to the tempo box and wishing I could tap a tempo right there and then.

**Comment by lgvr123** [comment]
> I agree this should be built into the new score wizard. I'm constantly going to the tempo box and wishing I could tap a tempo right there and then.

In the meantime you can use my TapTempo plugin. Definitely works in MU3.6. Should work in MU4.x

**Comment by joexcm** [comment]
Absolutely, thanks for the plugin!

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/395069
